nixos/modules/services/web-apps/keycloak.nix

```diff
@@ -272,6 +265,25 @@ in
         '';
       };
 
+      themeDevMode = {
+        enable = mkEnableOption ''
+          the theme devleopment mode. This is a quite huge performance deficit as caching of themes is disabled.
+
+          This requires `services.keycloak.themeDevMode.themeDirPath` to be set.
+          `service.keycloak.themes` is ignored when this option is true
+        '';
+
+        themesDirPath = mkOption {
```

Comment:
Another option: just add a so called: "out of store symlink" to `themes`, without having to change the module.